src/styles/styles.scss

```diff
@@ -179,16 +179,7 @@ p-panel {
 
 p-dialog {
     .ui-dialog {
-        width: 65% !important;
-
-        @include respond-to('ex-small') {
-            left: 17.5% !important;
-        }
-        @include respond-to('small') {
-            left: 17.5% !important;
-        }
-
-        top: 109px;
+        width: 65%;
```

Comment:
All of the modals were absolutely positioned to be a certain size and not collide with the navbar. This made it very difficult to create the full-screen modal, so I had to change this. We should take a look at all modals in the app to make sure they look ok.